### ISSUE TITLE
Forwarded notifications are muted despite being setup with a notification sound

### ISSUE THREAD

**Issue by martin-braun** [GrapheneOS/os_issue_tracker#1596]
If I forward notifications of a user profile to the current user profile and receive such a notification on the current user profile, the notification is visible, but the notification sound is not used. I long-pressed the notification and made sure that notification sounds are enabled for these types of notifications, but they remain muted (meaning they do not make any noise).

**Comment by thestinger** [comment]
This is just how it works right now: all the forwarded notifications appear in 1 channel on the receiving user.

**Comment by martin-braun** [comment]
@thestinger I see. For me personally, it would be no big deal if I could only use the same notification sound on all off-profile apps. The main problem for me is that no notification sound is played at all, despite setting it up.

**Comment by thestinger** [comment]
You need to configure it in the profile receiving notifications instead of the profiles sending them. Have you tried doing that?

**Comment by martin-braun** [comment]
@thestinger Yes. There is the default notification sound set in the profile for the app in question (it's WhatsApp on a secondary profile, installed through Aurora Store, but with all sandboxed Google Play things installed allow FCM to work) and its notifications would play sound as expected, if I'm on the same profile. It's only the forwarding to a different profile that will fail to play any notification sound. I set all default notification sounds to the same on all profiles ("Adara").

When I realized the issue on my primary profile where I got the forwarded notifications, I would long-press it to see its notification settings for some reason notification sound was disabled, so I enabled it and set the sound explicitly to "Adara" again, but the outcome did not change.

Can the issue not be reproduced?
